src/components/core/PivotTable.tsx

```diff
@@ -1165,10 +1147,23 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         this.updateStickyRowContent(scrollPosition);
     };
 
-    private preventHeaderResizerEvents = (event: Event) => {
+    private onContainerMouseDown = (event: MouseEvent) => {
         if (event.target && this.isHeaderResizer(event.target as HTMLElement)) {
             event.stopPropagation();
         }
+        this.isKeyMetaPressed = event.metaKey;
+        this.isKeyCtrlPressed = event.ctrlKey;
```

Comment:
please try to keep naming consistent (`metaKey` vs `keyMeta`)

Reply:
done